_data/simple-icons.json

```
@@ -8645,6 +8645,11 @@
             "hex": "0790C0",
             "source": "https://www.gnu.org/software/octave/"
         },
+        {
+            "title": "October CMS",
+            "hex": "DB6A26",
+            "source": "https://github.com/octobercms/october/blob/e3b455ad587282f0fbcb7763c6d9c3d000ca1e6a/modules/backend/assets/images/october-leaf.svg"
```

SVG icon available in the header of the website.

```suggestion
            "source": "https://octobercms.com"
```